### PR TITLE
Bugfix/issue template

### PR DESCRIPTION
**Pull request by alii-gator**
This PR is to update the Issue template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      label: Node Version
+      description: "Please provide the full Node version of your project. node -v"
+      placeholder: "v18.15.0"
+    validations:
+      required: true
+  - type: input
+    id: java
+    attributes:
+      label: JAVA Version
+      description: "Please provide the full Java version. java -version"
```
Comment (davidecorreu) [inline .github/ISSUE_TEMPLATE/bug-report.yml:53]: `java --full-version`

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+  - type: dropdown
+    id: tool
+    attributes:
+      label: Select the SuiteCloud tool where you encountered the issue
+      options:
+        - SuiteCloud CLI
+        - SuiteCloud Extension for VSCode
+        - SuiteCloud Unit Testing
```
Comment (davidecorreu) [inline .github/ISSUE_TEMPLATE/bug-report.yml:19]: Is it possible to remove the none option from the options? It appears in drop down if I "preview" this file.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    attributes:
+      label: Operating system
+      description: "Which operating system do you use? Please provide the version as well."
+      placeholder: "macOS Big Sur 11.5.2"
+    validations:
+      required: true  
+  - type: input
+    id: Node
+    attributes:
+      label: Node Version/ VSCode Version
```
Comment (davidecorreu) [inline .github/ISSUE_TEMPLATE/bug-report.yml:46]: Can you please add a blank space character after node version ?  `label: Node Version / VSCode Version`

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      label: Expected Output
+      description: |
+        Please provide the expected result/output
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+      required: false 
+  - type: textarea
+    attributes:
+      label: Anything else?
```
Comment (davidecorreu) [inline .github/ISSUE_TEMPLATE/bug-report.yml:100]: I'm fine as it is now, but what do you think about renaming this to `Extra relevant information`

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 3 above)
Comment (Montsant19) [reply .github/ISSUE_TEMPLATE/bug-report.yml:46]: Agree

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    validations:
+      required: true  
+  - type: input
+    id: Node
+    attributes:
+      label: Node Version/ VSCode Version
+      description: |
+        Please provide the full Node version of your project. node -v 
+           OR
+        Please provide VSCODE version in case you selected SuiteCloud Extension. This can be checked in "About VIsual Studio Code".
```
[the remark this answers is not in the document]
Comment (Montsant19) [reply .github/ISSUE_TEMPLATE/bug-report.yml:50]: Agree

**Review by davidecorreu — APPROVED** [review]

**Pull request merged**